src/cli/run_command.zig

```diff
@@ -1160,6 +1160,11 @@ pub const RunCommand = struct {
             }
         }
 
+        // If the script is not present and --if-present is passed, exit with 0
+        if (ctx.runtime_options.if_present) {
```

Comment:
This should only happen when `comptime log_errors` to account for both `bun run --if-present` and `bun --if-present`